src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -719,6 +765,25 @@ private function createCart(
         return $order;
     }
 
+    private function createCustomer(string $email): CustomerInterface
+    {
+        /** @var CustomerInterface $customer */
+        $customer = $this->customerFactory->createNew();
+        $customer->setEmail($email);
+        $customer->setFirstName('John');
+        $customer->setLastName('Doe');
+
+        return $customer;
+    }
+
+    private function createOrProvideCustomer(string $email): CustomerInterface
+    {
+        /** @var CustomerInterface $customer */
+        $customer = $this->customerRepository->findOneBy(['email' => $email]);
+
+        return ($customer !== null) ? $customer : $this->createCustomer($email);
```

Comment:
```suggestion
        return $customer ?? $this->createCustomer($email);
```
should work the same